src/browser/settings.jsx

```diff
@@ -56,6 +71,14 @@ var SettingsPage = React.createClass({
       var currentWindow = remote.getCurrentWindow();
       currentWindow.setAutoHideMenuBar(config.hideMenuBar);
       currentWindow.setMenuBarVisibility(!config.hideMenuBar);
+
+      var autostart = this.state.autostart;
+      appLauncher.isEnabled().then(function(enabled) {
+        if (enabled && !autostart)
```

Comment:
Please use brackets. Not only readability for me, `npm run prettify` incorrectly beautified this.